Add 2025_Nordfors_MedievalFinland

smpeltola opened the pull request:
### PR Checklist for a new package submission

- [X] The package does not exist already in the community archive, also not with a different name.
- [X] The package title in the `POSEIDON.yml` conforms to the general title structure suggested here: `<Year>_<Last name of first author>_<Region, time period or special feature of the paper>`, e.g. `2021_Zegarac_SoutheasternEurope`, `2021_SeguinOrlando_BellBeaker` or `2021_Kivisild_MedievalEstonia`.
- [X] The package is stored in a directory that is named like the package title.

***

- [ ] Samples that already have been published previously, and got re-analysed (e.g. re-sequenced) for the now packaged publication, have a modified `Poseidon_ID` of the form `<Original Poseidon_ID>_<Initials of the main author>_<Year>`. Re-analysed versions of `I1685` (Lazaridis et al. 2016) should, for example, be assigned the IDs `I1685_IL22` (Lazaridis et al. 2022) and `I1685_IL25` (Lazaridis et al. 2025).

***

- [X] The package is complete and features the following elements:
  - [X] Genotype data in binary PLINK format (not EIGENSTRAT format).
  - [X] Genotype has been provided by the original authors of the publication describing the data.
  - [X] A `POSEIDON.yml` file with not just the file-referencing fields, but also the following meta-information fields present and filled: `poseidonVersion`, `title`, `description`, `contributor`, `packageVersion`, `lastModified` (see [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/POSEIDON_yml_fields.tsv) for their definition)
  - [X] A reasonably filled `.janno` file (for a list of available fields look [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv) and [here](https://www.poseidon-adna.org/#/janno_details) for more detailed documentation about them).
  - [X] A `.bib` file with the necessary literature references for each sample in the `.janno` file.
- [X] Every file in the submission is correctly referenced in the `POSEIDON.yml` file and there are no additional, supplementary files in the submission that are not documented there.
- [X] Genotype data, `.janno` and `.bib` file are all named after the package title and only differ in the file extension.
- [X] The package version in the `POSEIDON.yml` file is `1.0.0`.
- [X] The `poseidonVersion` of the package in the `POSEIDON.yml` file is set to the latest version of the [Poseidon schema](https://github.com/poseidon-framework/poseidon-schema/releases).
- [X] The `POSEIDON.yml` file contains the corresponding checksums for the fields `genoFile`, `snpFile`, `indFile`, `jannoFile` and `bibFile`.
- [X] There is either no `CHANGELOG` file or one with a single entry for version `1.0.0`.

***

- [X] The `Publication` column in the `.janno` file is filled and the respective `.bib` file has complete entries for the listed mentioned keys.
- [X] The `.janno` file does not include any empty columns or columns only filled with `n/a`.
- [X] The order of columns in the `.janno` file adheres to the standard order as defined in the Poseidon schema [here](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv).
- [X] The `.janno` and the `.ssf` files are not fully quoted, so they only use single- or double quotes (`"..."`, `'...'`) to enclose text fields where it is strictly necessary (i.e. their entry includes a TAB).

***

- [X] The package passes a validation with `trident validate --fullGeno`.

***

- [X] Large genotype data files are properly tracked with Git LFS and not directly pushed to the repository. For an instruction on how to set up Git LFS please look [here](https://www.poseidon-adna.org/#/archive_submission_guide?id=submitting-the-package). If you accidentally pushed the files the wrong way you can fix it with `git lfs migrate import --no-rewrite path/to/file.bed` (see [here](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-migrate.adoc#import-without-rewriting-history)).

stschiff commented:
Thanks for the submission!

nevrome commented:
@EleftheriaOrfanou kindly offered to review this package! Thank you very much!
The review instructions are available here: https://www.poseidon-adna.org/#/archive_reviewer_guide
Feel free to ask in case something about this process is unclear.

EleftheriaOrfanou commented:
Hi Sanni, I just checked the package. Overall, it looks good! However: 

1. The janno file contains unreadable characters in columns such as location, site, and publication. All characters in .janno files must be UTF-8 encoded. 
2. In the janno file, I could not locate the group name in the original publication, so I am not sure if it matches. Could you double-check that?
3. In the bib file, there is the month missing for the first publication, but I'm not sure if it is necessary. 

Let me know if you have any questions!

nevrome commented:
Thanks for the review, @EleftheriaOrfanou!

1. I checked the encoding and can't find any issues. Maybe this is a special property of your system or the font you're using, @EleftheriaOrfanou? Maybe the font was just lacking these characters?
2. `Group_Name` is a list column and allows multiple entries. Only the first must match the .fam file. I suggest we keep the more informative ones in the primary position, but add the original ones from the paper as secondary entries? Or the other way round. I leave that up to you, @smpeltola. `trident forge` considers both, actually.
3. :white_check_mark:

smpeltola commented:
Alright, I adjusted the group names as @nevrome suggested, the second label now matches the publication exactly. I also added the missing month in the bib entry.

nevrome commented:
Good! @EleftheriaOrfanou and I also figured out what caused the encoding problems for her in the meantime. The issue was in the way MS Excel was reading the data.

The paper is [out now](https://www.sciencedirect.com/science/article/pii/S2589004225013471), right? Congrats!! :tada:

So maybe you could complete the bibtex entry and I can merge the package then, @smpeltola.

smpeltola commented:
Thanks! The journal has only published (an ugly) pre-proof version online, so it doesn't have all the bibliographical information yet. But I suppose the paper is technically published if you want to merge already.

nevrome commented:
Excellent! I'll merge it.
